recipes/cppyy-cling/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Much of this file (and the entire recipe in fact) has been taken from the
+# root-feedstock.
+
+export STDCXX=17
+export VERBOSE=1
+
+# Manually set the deployment_target
+# May not be very important but nice to do
+OLDVERSIONMACOS='${MACOSX_VERSION}'
+sed -i -e "s@${OLDVERSIONMACOS}@${MACOSX_DEPLOYMENT_TARGET}@g" src/cmake/modules/SetUpMacOS.cmake
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_AR=${GCC_AR}")
+    CMAKE_PLATFORM_FLAGS+=("-DCLANG_DEFAULT_LINKER=${LD_GOLD}")
+    CMAKE_PLATFORM_FLAGS+=("-DDEFAULT_SYSROOT=${PREFIX}/${HOST}/sysroot")
+    CMAKE_PLATFORM_FLAGS+=("-DRT_LIBRARY=${PREFIX}/${HOST}/sysroot/usr/lib/librt.so")
+
+    # Hide symbols from LLVM/clang to avoid conflicts with other libraries
+    for lib_name in $(ls $PREFIX/lib | grep -E 'lib(LLVM|clang).*\.a'); do
+        export CXXFLAGS="${CXXFLAGS} -Wl,--exclude-libs,${lib_name}"
```

Comment:
Shouldn't this be LDFLAGS instead of CXXFLAGS? Changing this would probably fix your warning

Reply:
This is from root's build script. Let's try.

Reply:
Then again, this won't help on macOS.

Reply:
Actually, the warning now only shows on macOS, so I guess there's no need to change this.